src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -69,6 +69,7 @@ def load_arguments(self, _):
         c.argument('name', options_list=['--name', '-n'], help="Name of the new app service plan", completer=None)
         c.argument('sku', arg_type=sku_arg_type)
         c.argument('is_linux', action='store_true', required=False, help='host webapp on Linux worker')
+        c.argument('hyper_v', action='store_true', required=False, help='Preview - host webapp on Windows container')
```

Comment:
the convention is `[PREVIEW]....`

Reply:
Seems like you didn't push your change

Reply:
Hi @adewaleo . I did. I can see the change when I look to the commit link: https://github.com/Azure/azure-cli/pull/7390/commits/a35af1243c228f3b98c16a47c3f643803a48b84e but somehow it doesn't refresh the main PR page. Let me see what is wrong.

Reply:
I see what happened. I fixed it in webapp create but missed it in appservice plan create. I fixed it now.